src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssPropertiesAndValues = sourceString.split(';');
+  const result = {};
+
+  cssPropertiesAndValues.forEach((propertyValuePair) => {
+    const trimmedItem = propertyValuePair.trim();
+
+    if (trimmedItem.length < 4) {
+      return;
+    }
+
+    const [key, value] = trimmedItem.split(':');
+
+    if (key && value) {
+      result[key.trim()] = value.trim();
+    }
+  });
```

Comment:
You should avoid using forEach for mutating an existing object. Instead, consider using `map` to create an array of objects, and then `Object.assign` or object spread syntax to merge them into a single object. This approach is more functional and avoids direct mutation of state.